frontend/src/index.pug

```diff
@@ -132,6 +132,13 @@ html
               label
                 input(type="checkbox", v-model="filterBreakdown").mui-checkbox.summary-picker__checkbox
                 span breakdown by file format
+        .error-message-box(v-if="Object.entries(errorMessages).length")
+          .error-message-box__close-button(onclick="this.parentElement.style.display='none'") &times;
```

Comment:
Might be better to create a method in main.js to hide the box rather than reference the element in html